tests/unittests/parsing/VisitorTests.cpp

```diff
@@ -745,14 +783,30 @@ TEST_CASE("Visit all file") {
     Compilation compilation;
     compilation.addSyntaxTree(*tree);
 
-    int count = 0;
-    compilation.getRoot().visit(makeVisitor([&](auto& v, auto& elem) {
```

Comment:
It would be nice to keep a `makeVisitor` call somewhere, since that specific method has been broken in the past.